Geometry/HGCalCommonData/src/HGCalCellUV.cc

```diff
@@ -266,3 +267,50 @@ std::pair<int, int> HGCalCellUV::cellUVFromXY4(double xloc,
                                   << uv.first << ":" << uv.second;
   return uv;
 }
+
+std::pair<int32_t, int32_t> HGCalCellUV::cellUVFromXY1(
+    double xloc, double yloc, int32_t placement, int32_t type, int32_t partial, bool extend, bool debug) {
+  std::pair<int, int> uv = HGCalCellUV::cellUVFromXY1(xloc, yloc, placement, type, extend, debug);
+  int u = uv.first;
+  int v = uv.second;
+  if (partial == HGCalTypes::WaferLDTop) {
+    if (u > 7) {
```

Comment:
6, 7, and 10 look like magic numbers in this code. They should probably be replaced by constants.